docs/content/documentation/templates/overview.md

```diff
@@ -258,6 +258,19 @@ integrity](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Int
 
 Do note that subresource integrity is typically used when using external scripts, which `get_file_hash` does not support.
 
+### `get_hash`
+
+Returns the hash digest (SHA-256, SHA-384 or SHA-512) of a string literal.
+
+It can take the following arguments:
+- `literal`: mandatory, the value to be hashed
+- `sha_type`: optional, one of `256`, `384` or `512`, defaults to `384`
+- `base64`: optional, `true` or `false`, defaults to `true`. Whether to encode the hash as base64
+
+```jinja2
+{{/* get_hash(literal="Hello World", sha_type=256) */}}
+```
+
```

Comment:
Let's merge `get_file_hash` and `get_hash` into a single function `get_hash` that takes either a literal or a path

Reply:
Okay, I’ll work on it tomorrow (30/08). I guess I’ll have to make the `get_hash` function into a `GetHash` struct with a `TeraFn` impl to make it work.

Are you sure about the deprecation of `get_file_hash` ? That’s quite a big breaking change (even if I agree with the move).